FSJ_django20_project/FSJ/forms_award.py

```diff
@@ -0,0 +1,31 @@
+from .models_award import Award
+from django.forms import ModelForm, CheckboxSelectMultiple
+
+class AwardForm(ModelForm):
+
+    class Meta:
+        model = Award
+        exclude = ()
+        #uncomment this out when awards are ready for multiple years of study
+        # fields = ('award_name', 'description', 'value', 'programs', 'years_of_study', 'deadline', 'documents_needed', 'is_active')
+        # widgets = {
+        #     'years_of_study': CheckboxSelectMultiple(),
+        # }
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Defining an init method that simply calls super is unnecessary since the default would be to call the init method of the parent class in the absence of an init on the child class